xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -1422,7 +1422,10 @@ bool CGUIDialogVideoInfo::UpdateVideoItemTitle(const std::shared_ptr<CFileItem>&
   else if (mediaType == MediaTypeSeason)
   {
     database.GetSeasonInfo(iDbId, detail);
-    title = detail.m_strSortTitle;
+    if (detail.m_strSortTitle.empty())
```

Comment:
ternary operator would make this more concise

Reply:
done